contracts/yield-source/ATokenYieldSource.sol

```diff
@@ -9,23 +9,25 @@ import "@openzeppelin/contracts-upgradeable/math/SafeMathUpgradeable.sol";
 import "@openzeppelin/contracts-upgradeable/token/ERC20/ERC20Upgradeable.sol";
 import "@openzeppelin/contracts-upgradeable/token/ERC20/IERC20Upgradeable.sol";
 import "@openzeppelin/contracts-upgradeable/token/ERC20/SafeERC20Upgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/utils/ReentrancyGuardUpgradeable.sol";
```

Comment:
Using Upgradable due to ease of solidity compiler compatibility - non-upgradeable requires ^0.5.0 (and other contracts are upgradeable)